participant-manager-datastore/participant-manager-service/src/main/java/com/google/cloud/healthcare/fdamystudies/config/WebSecurityConfig.java

```diff
@@ -22,10 +22,18 @@
 
   @Autowired AppPropertyConfig appConfig;
 
+  /**
```

Comment:
Please update file copyright to `2020-2021`

Reply:
Updated